File: soal3.py
# Menentukan berapa data yang ingin diinput
jumlah_data = int(input("Masukkan jumlah data yang ingin dicek: "))

# List untuk menyimpan hasil
data_list = []

for i in range(jumlah_data):
    print(f"\nData ke-{i+1}")
    kuota = int(input("Masukkan kuota maksimal: "))
    terdaftar = int(input("Masukkan jumlah peserta terdaftar: "))

    # Conditional logic
    if terdaftar > kuota:
        status = "Pendaftaran ditolak"
    elif terdaftar == kuota:
        status = "Pendaftaran ditutup"
    elif terdaftar >= kuota - 3:
        status = "Hampir penuh"
    else:
        status = "Pendaftaran diterima"

    # Simpan ke list
    data_list.append((kuota, terdaftar, status))


# Output tabel
print("\nKuota\tTerdaftar\tOutput")
for data in data_list:
    print(f"{data[0]}\t{data[1]}\t\t{data[2]}")

File: soal3.js
const prompt = require("prompt-sync")({ sigint: true });

// Menentukan berapa data yang ingin diinput
let jumlahData = parseInt(prompt("Masukkan jumlah data yang ingin dicek: "));

// Array untuk menyimpan hasil
let dataList = [];

for (let i = 0; i < jumlahData; i++) {
  console.log(`\nData ke-${i + 1}`);
  
  let kuota = parseInt(prompt("Masukkan kuota maksimal: "));
  let terdaftar = parseInt(prompt("Masukkan jumlah peserta terdaftar: "));
  
  let status;

  // Conditional logic
  if (terdaftar > kuota) {
    status = "Pendaftaran ditolak";
  } else if (terdaftar === kuota) {
    status = "Pendaftaran ditutup";
  } else if (terdaftar >= kuota - 3) {
    status = "Hampir penuh";
  } else {
    status = "Pendaftaran diterima";
  }

  // Simpan ke array
  dataList.push({ kuota, terdaftar, status });
}

// Output tabel
console.log("\nKuota\tTerdaftar\tOutput");
dataList.forEach(data => {
  console.log(`${data.kuota}\t${data.terdaftar}\t\t${data.status}`);
});
